Mathlib/Data/Complex/Basic.lean

```diff
@@ -638,6 +638,11 @@ theorem normSq_one : normSq 1 = 1 :=
   normSq.map_one
 #align complex.norm_sq_one Complex.normSq_one
 
+@[simp]
+theorem normSq_ofNat (n : ℕ) [n.AtLeastTwo] :
+    normSq (no_index (OfNat.ofNat n : ℂ)) = OfNat.ofNat n * OfNat.ofNat n := by
```

Comment:
I think this might be nicer as
```suggestion
    normSq (no_index (OfNat.ofNat n : ℂ)) = OfNat.ofNat n ^ 2 := by
```

Reply:
This would require changing the rhs of at least `normSq_ofReal` to preserve confluence:
```lean
example : Complex.normSq (5 : ℝ) = 5 * 5 := by
  simp only [normSq_ofReal]

example : Complex.normSq (5 : ℝ) = 5 ^ 2 := by
  simp only [ofReal_ofNat, normSq_ofNat]
```

Reply:
Fair enough